src/routes/index.jsx

```diff
@@ -5,13 +5,15 @@ import Header from '../components/Header';
 import Footer from '../components/Footer';
 import Home from '../components/Home';
 import Login from '../components/Login';
+import SocialAccount from '../components/SocialAccount';
 
 const Routes = () => (
   <BrowserRouter>
     <div className="App">
       <Header />
       <Route exact path="/" component={Home} />
       <Route path="/login" component={Login} />
+      <Route path="/social-login" component={SocialAccount} />
```

Comment:
@cob04, you do not need a route for this, you could use the modal. If you need to deviate, notify the team so we discuss it.

Reply:
thank you, this has been resolved